deploy/list-connectors.ts

```diff
@@ -0,0 +1,108 @@
+import { glob } from "tinyglobby";
+import fs from "node:fs/promises";
+import { join } from "node:path";
+import Module from 'node:module';
+
+const ROOT_DIR = join(__dirname, "..");
+
+const IGNORE_PACKAGE_LIST = ["api", "backend", "shared"];
+
+// @ts-ignore
+const originalLoad = Module._load;
+
+// mocuk the module loading to prevent errors when a module is not found
+// @ts-ignore
+Module._load = function(path: string) {
+  // mock typia
+  if (path.includes("typia")) {
+    // prevent mocking the typia/lib/transform module
+    if(path.includes("typia/lib/transform")) {
+      return originalLoad.apply(this, arguments);
+    }
+    console.warn(`Warning: Mock typia module`);
+    return {
+      createIs: () => {}
+    };
+  }
+  try {
+    return originalLoad.apply(this, arguments);
+  } catch (error) {
+    // ignore integral package import errors
+    // @ts-ignore
+    if(path.includes("@wrtnlabs")) {
+      console.warn(`Warning: Module "${path}" not found. Using mock instead.`);
+      return {}
+    }
+    // mock typia
+    return originalLoad.apply(this, arguments);
+  }
+};
```

Comment:
Dirty hack
Hopefully we wanna avoid it but since we don't have enough time before releases we have to do that

Reply:
wow. you're genius. take it time and let's solve this problem.